src/actions/GridActions.js

```diff
@@ -238,7 +238,11 @@ export const doRemoteSort = ({
         );
 
         if (typeof dataSource === 'function') {
-            return dataSource({pageIndex}, {}, sortParams).then((response) => {
+            return dataSource(
+                {pageIndex},
+                filterFields,
+                sortParams
```

Comment:
Updated this to include group pageIndex and pageSize as a pagingParams object to match pager's setPageIndexAsync's pattern: dataSource({ pageIndex, pageSize }, filterFields, sortParams)